src/vulkan/buffer.h

```diff
@@ -34,7 +34,10 @@ class Buffer : public Resource {
   Result CreateVkBufferView(VkFormat format);
   VkBufferView GetVkBufferView() const { return view_; }
 
-  // TODO(jaebaek): Determine copy all or partial data
+  // If this buffer is host accessible, this method does nothing.
+  // Otherwise, this method only records the command for copying
+  // this buffer to the host accessible buffer. The actual
```

Comment:
.. "this buffer to the host accessible buffer".  ??  Isn't that the wrong way around?

Reply:
Also, compared to CopyToHost, why is this method different regarding buffers marked as COHERENT?

Reply:
Updated.